hexrd/ui/load_panel.py

```diff
@@ -234,14 +234,16 @@ def enable_aggregations(self, row, column):
         total_frames = np.sum(self.total_frames)
         if total_frames - self.empty_frames < 2:
             enable = False
-        self.ui.darkMode.setEnabled(enable)
         self.ui.aggregation.setEnabled(enable)
+        for i in range(4):
+            self.ui.darkMode.model().item(i).setEnabled(enable)
 
         if not enable:
             # Update dark mode settings
-            num_dets = len(HexrdConfig().detector_names)
-            self.state['dark'] = [5 for x in range(num_dets)]
-            self.ui.darkMode.setCurrentIndex(5)
+            if self.ui.darkMode.currentIndex() < 4:
```

Comment:
Lets use the constants, that we have defined [here](https://github.com/HEXRD/hexrdgui/blob/master/hexrd/ui/constants.py#L9-L14)

Reply:
Fixed